ui/src/modules/autoconsent/views/home.js

```diff
@@ -27,68 +27,68 @@ function onConfirm(type) {
 }
 
 export default define({
+  [router.connect]: { stack: [Confirm] },
   tag: 'ui-autoconsent-home-view',
   scope: defaultChoice === 'all' ? 'all' : 'selected',
-  content: () =>
-    html`
-      <template layout="column margin:3 gap:4">
-        <div layout="column items:center gap">
-          <ui-text type="display-s" layout="block:center">
-            TIRED OF COOKIE POPUPS?
-          </ui-text>
-          <ui-text layout="block:center">
-            Let Ghostery be your complete privacy advocate and reject all popups
-            and tracking for you, or do it yourself!
-          </ui-text>
+  content: ({ scope }) => html`
+    <template layout="column margin:3 gap:4">
```

Comment:
it kinda suck we had to update the indentation as it makes the diff not readable. 

To avoid it in the future, could we have big `html` blocks with no indentation?

Reply:
i think it was another type of issue, as usually, we should start with one line ending with `html`, but this time it was different, so I changed that for the consistency.